src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -709,6 +732,8 @@ private static Options buildOptions() {
         options.addOption(OPTION_C_NAME, true, "Sets the check configuration file to use.");
         options.addOption(OPTION_O_NAME, true, "Sets the output file. Defaults to stdout");
         options.addOption(OPTION_P_NAME, true, "Loads the properties file");
+        options.addOption(OPTION_S_NAME, true, "Generates basic suppression xpath");
```

Comment:
The other similar options say "Print ....".
This description should follow suite.
"Print xpath suppressions at the file's line and column position"

Reply:
Done